config/secrets.example.yml

```diff
@@ -68,3 +69,8 @@ production:
     :enable_starttls_auto: true
   google_analytics_code: <%= ENV["PRODUCTION_GANAL_CODE"] %>
   google_maps_api_key:   <%= ENV["PRODUCTION_GMAPS_KEY"] %>
+  database:
+    :host: db
+    :name: taxila
+    :username: taxila
+    :password: password
```

Comment:
this hardcodes the production credentials which are also set in env.sample. Not the best way of doing it, please remove here (use ENV vars, but I think that already works)